Sources/TABResourceLoader/Extensions/GlobalFunctions.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  GlobalFunctions.swift
+//  TABResourceLoader
+//
+//  Created by Luciano Marisi on 11/02/2017.
+//  Copyright © 2017 Luciano Marisi. All rights reserved.
+//
+
+import Foundation
+
+/// Merges two [String: String] dictionaries. For common keys, the values of the right dictionary take precedence
+///
+/// - Parameters:
+///   - left:  The dictionary to use as a base
+///   - right: The dictionary used to append to the base, if a key from this dictionary exists in the base,
+///            the value from this dictionary will be used
+/// - Returns: Returns the merged dictionary or nil if both dictionary are nil
+func merge(_ left: [String: String]?, _ right: [String: String]?) -> [String: String]? {
```

Comment:
i presume this is covered by existing unit tests? why not add explicit tests to cover this function? also is there no way you can add it in an extension on `Dictionary where Self.Key == String && Self.Value == String` or something? maybe it won't work. but tests though...